plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -1337,7 +1339,7 @@ private void storePartitionColumnStatistics(HiveIdentity identity, String databa
         }
         Map<String, HiveType> columnTypes = partitionWithStatistics.getPartition().getColumns().stream()
                 .collect(toImmutableMap(Column::getName, Column::getType));
-        setPartitionColumnStatistics(identity, databaseName, tableName, partitionName, columnTypes, columnStatistics, statistics.getBasicStatistics().getRowCount());
+        setPartitionColumnStatistics(identity, databaseName, tableName, partitionName, columnTypes, columnStatistics, statistics.getBasicStatistics().getRowCount(), NO_ACID_TRANSACTION);
```

Comment:
Add a comment why NO_ACID_TRANSACTION and what will happen if we hit here for a transactional table.

Reply:
Added.